### PR TITLE
Add bio for GitHub Student

### PR DESCRIPTION
**Pull request by githubstudent**
This was requested in issue #2 — adding a bio for GitHub Student as an example.

### CODE REVIEW

**Comment by githubstudent** [comment]
@githubteacher can you please review this pull request?

File: githubstudent.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+* **Name:** GitHub Student
```
Comment (githubteacher) [inline githubstudent.md]: Maybe we should start this with a headline?

File: githubstudent.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+* **Login:** githubstudent
```
Comment (githubteacher) [inline githubstudent.md]: Formatting: this should really be `@githubstudent`

**Comment by githubteacher** [comment]
Overall, this is pretty good. If you address those two bits above, then :+1:

**Pull request merged**